File: solutions/easy/Move Zeroes.java
// Title: Move Zeroes
            // Difficulty: Easy
            // Language: Java
            // Link: https://leetcode.com/problems/move-zeroes/

class Solution {
    public void moveZeroes(int[] nums) {
       int i =0;
       int j =1;
       while(j<nums.length){
            if(nums[i]==0){
                if(nums[j]==0) i--; 
                else{
                    int temp = nums[j];
                    nums[j] = nums[i];
                    nums[i] = temp;
                }
                }
                i++;
                j++;
        }
    }
}

File: solutions/easy/Move Zeroes.c
// Title: Move Zeroes
            // Difficulty: Easy
            // Language: C
            // Link: https://leetcode.com/problems/move-zeroes/

     int nonZero=0;
void moveZeroes(int* nums, int numsSize) {
     int i =0;
     for(i =0 ;i< numsSize;i++){
        if(nums[i]!=0){
            nums[nonZero]= nums[i];
            nonZero++;
        }
     }
        for(i = nonZero;i<numsSize;i++)
            nums[i]=0;
     }

    

